css/styles.css

```diff
@@ -0,0 +1,67 @@
+* {
+  font-family: "Proza Libre", sans-serif;
+}
+
+label {
+  font-weight: 600;
+}
+
+button {
+  background-color: #502274;
+  color: white;
+  padding: 6px 12px;
+  margin-top: 10px;
+  margin-bottom: 28px;
+  border: none;
+  border-radius: 3px;
+}
+
+input {
+  padding: 8px;
+  margin: 4px 0 14px;
+}
+
+textarea {
+  height: 150px;
+}
+
+input:focus,
+textarea:focus {
+  outline: none;
+  border: 2px solid #502274;
+}
+
+img {
+  width: 100%;
+  height: auto;
+  margin-bottom: 20px;
+  object-fit: contain;
+}
+
+.save {
+  text-align: right;
+}
+
+.banner {
+  max-width: 575px;
+  margin: 0 auto;
+  font-size: 24px;
+  padding: 0 1.5rem;
+  background-color: #502274;
+  color: white;
+}
+
+.user-input {
+  width: 100%;
+  border-radius: 3px;
+  border: 1px solid lightgray;
+}
+
+.page-color {
+  background-color: rgb(245, 245, 245);
+}
+
+.container2 {
+  max-width: 100%;
+  background-color: #502274;
+}
```

Comment:
You shouldn't make a second `container` class, just use the original one and add another class for the background color in addition to the `container` class.